src/components/ChattingBubble/index.tsx

```diff
@@ -0,0 +1,58 @@
+import type { HTMLAttributes, ReactElement } from 'react'
+import styled from '@emotion/styled'
+
+export interface ChattingBubbleProps extends HTMLAttributes<HTMLDivElement> {
+  type: 'send' | 'receive'
+  children: string
+}
+
+interface StyledBubbleProps {
+  type: 'send' | 'receive'
+  isSend: boolean
+}
+
+export const ChattingBubble = ({
+  type,
+  children,
+  ...props
+}: ChattingBubbleProps): ReactElement => {
+  const newLineRegex = /\s/gi
+  const blankChatLength = children?.length
+  const noBlankchatLength = children?.replace(newLineRegex, '').length
```

Comment:
```suggestion
  const noBlankChatLength = children?.replace(newLineRegex, '').length
```
대문자로 구문지으면 좋을 것 같습니다!